### ISSUE TITLE
GetKeyword method not working as expected.

### ISSUE THREAD

**Issue by mirkoSekulic** [gregsdennis/json-everything#504]
### Nuget Package

JsonSchema.Net

### Package Version

5.1.3

### Operating System

Windows

### .Net Target (if relevant, please specify the version in the description)

.Net (5 or after)

### A clear and concise description of the bug

Seems like GetKeyword method stoped working as expected. When the keyword is not present it throws an error, but in the description of the method it states following: "Returns: The keyword if it exists; otherwise null.".

Error occures due to dictionary implementation under the hood, and it throws KeyNotFoundException exception.
Here is the error message:
"The given key 'allOf' was not present in the dictionary."

### Please add test code or steps to reproduce the behavior

Load the json schema that doesn't contain allOf keyword and call schema.GetKeyword<AllOfKeyword>(); method.

### What did you expect?

Method to return null if keyword is not present

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by gregsdennis** [comment]
Nothing has changed about this method since it was [introduced](https://github.com/gregsdennis/json-everything/commit/86d21bd8d93413b4473af26fe8dbef7bae5c074e).  Is this report merely about the documentation saying it should return null?

**Comment by mirkoSekulic** [comment]
@gregsdennis Sorry, this was my bad. 
When we used old version of the lib, we've made our extension with the same signature as you have now. When I tried to upgrade to the latest version I missed that it was it was newly introduced method. I forgot that we had our implementation in the first place when I read the description of the method. Anyway, I guess it'd be nice to fix the documentation.
Will close the task. Thanks
